wp-content/themes/core/blocks/core/button/style.pcss

```diff
@@ -2,6 +2,10 @@
  * Styles specific to this block
  */
 
+.wp-block-buttons {
+	gap: var(--wp--preset--spacing--30);
+}
+
```

Comment:
Is the button**S** block technically different from the button block? I'd hate to spin up all the code needed for just this little css, but...I'd also like to make sure we're consistent with how Core divides up the blocks. Thoughts?

Reply:
Technically, yes, buttons vs button are different blocks.

Given the buttons block just has layout styles correlating to block settings, and I can't see a future where we'd really need to customize buttons block styles more than setting the gap value I decided to just add it in to the "button" block.

I'm all for consistencly but, it does seem like overkill (to me) to set up all the code just for the one rule. It also is easier to skim the directory and not have to mentally decide which one to open/edit if there's just the one.